### ISSUE TITLE
[Docs] Sample TS + AWS SAM projects that people can explore

### ISSUE THREAD

**Issue by verzac** [verzac/typescript-aws-sam#6]
Hey everyone! Thanks for coming across this project.

TL:DR; does anyone know of any fancy schmancy TS + AWS SAM project that they'd like to chuck in the docos?

So, I just realised something: this project is starting to get a lot of tractions (by my standard, which isn't much lmao). 

One thing that struck me was that I said that *this project structure is substandard*. However, I only gave people my project as an alternative, which is only one way to do things (because AWS SAM is an infrastructure framework, not a development framework, so you can literally bring any tooling you want into your dev workflow).

If you guys have come across any cool serverless project which combine TS + AWS SAM, do feel free to make a PR into `README.md`'s "Next Steps" section, or create a new section underneath it called "Sample Projects".
